src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -404,6 +406,32 @@ def _post_setup_success(self, update_revision=True):
     if instrumented_library_paths:
       self._patch_rpaths(instrumented_library_paths)
 
+  def _emmit_build_age_metric(self, gcs_path):
+    """Emmits a metric to track the age of a build."""
+    try:
+      last_update_time = storage.get(gcs_path).get('updated')
+      if isinstance(last_update_time, str):
+        # storage.get returns two different types for the updated field:
```

Comment:
we should fix this instead. Can you file a bug and add a TODO?

Reply:
https://github.com/google/clusterfuzz/issues/4357
Opened and assigned to myself